source/modules/MEI.xml

```diff
@@ -3071,6 +3071,31 @@
       </rng:data>
     </content>
   </macroSpec>
+  <macroSpec ident="data.PGFUNC" module="MEI" type="dt">
+    <desc xml:lang="en">Page header and footer function; a value that defines the function (i.e., the placement) of the header or the footer.</desc>
+    <content>
+      <valList type="closed">
+        <valItem ident="all">
+          <desc xml:lang="en">Header or footer for all the pages, including first and last unless provided.</desc>
```

Comment:
```suggestion
          <desc xml:lang="en">Header or footer for all pages, including the first and last page, unless provided.</desc>
```

Reply:
"unless provided" is unclear to me. In the sense of "unless declared otherwise"? Meaning that an additional declaration of "first" or "last" overrides the value for "all" in the first and last page?

Reply:
`Header or footer for all pages, including the first and the last page, unless a page header or footer for the first or the last page is provided.`
Does it work for you?

Reply:
Yes, much clearer for me, thanks!